src/test/ArraysTable.fbs

```diff
@@ -0,0 +1,53 @@
+struct Point3b {
+  xyz:[byte:3];
+}
+struct Point3s {
+  xyz:[short:3];
+}
+struct Point3i {
+  xyz:[int:3];
+}
+struct Point3l {
+  xyz:[long:3];
+}
+struct Point3f {
+  xyz:[float:3];
+}
+struct Point3d {
+  xyz:[double:3];
+}
+
+struct Mat3x3b {
+  cols:[Point3b:3];
+}
+struct Mat3x3s {
+  cols:[Point3s:3];
+}
+struct Mat3x3i {
+  cols:[Point3i:3];
+}
+struct Mat3x3l {
+  cols:[Point3l:3];
+}
+struct Mat3x3f {
+  cols:[Point3f:3];
+}
+struct Mat3x3d {
+  cols:[Point3d:3];
+}
+
```

Comment:
* Array of enums

Reply:
😮 didn't realize that was allowed. I guess enums are "just" a scalar type under the hood.

Reply:
Sadly I was unable to add this to the test because flatc crashes trying to generate the TS code. I'll probably file a ticket upstream.

Reply:
https://github.com/google/flatbuffers/issues/8311